features/project/source/browse_files.feature

```diff
@@ -133,6 +143,16 @@ Feature: Project Source Browse Files
     Then I am redirected to the files URL
     And I don't see the ".gitignore"
 
+  @javascript
+  Scenario: I can replace file and commit
+    Given I click on ".gitignore" file in repo
+    And I see the ".gitignore"
+    And I click on "Replace"
+    When I replace it with "LICENSE"
+    And I fill the commit message
+    And I click on "Replace file"
+    Then I am redirected to the ".gitignore"
+
```

Comment:
@tsigo @stanhu I attach one test image for clear.
![chinesereminder1](https://cloud.githubusercontent.com/assets/7370334/8353498/e169286a-1b3e-11e5-91b8-a89e5002a65b.png)